gensrc/thrift/Opcodes.thrift

```diff
@@ -88,11 +88,6 @@ enum TExprOpcode {
     MATCH_ANY,
     MATCH_ALL,
     MATCH_PHRASE,
-    MATCH_ELEMENT_EQ,
```

Comment:
Do not delete fields in thrift, maybe cause compability issue

Reply:
mark them as deprecated

Reply:
make sense

Reply:
make sense